src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -87,7 +87,7 @@ object FunctionRegistry {
             None
         }
 
-        anyFailAllFail[Array, Option[(Int, Transformation[Any, Any])]](conversions)
```

Comment:
Are there other places it is used that  you can clean up (like the change I pushed yesterday)?